client/flutter/lib/components/arrow_button.dart

```diff
@@ -11,15 +11,16 @@ class ArrowButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return FlatButton(
-      shape: StadiumBorder(),
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(35)),
```

Comment:
isn't `StadiumBorder` the correct shape?

Reply:
This doesn't change the shape when the text is on one line.

With large text the button turns almost into a circle with `StadiumBorder`, and just looks weird:

![Simulator Screen Shot - iPhone 5s - 2020-04-10 at 00 17 40](https://user-images.githubusercontent.com/756862/78941630-b9ef5200-7ac0-11ea-82a3-40b1007fee7a.png)

Reply:
Ah, cool.